spot_rl_experiments/spot_rl/skills/atomic_skills.py

```diff
@@ -33,49 +35,196 @@
 # Import core classes
 from spot_wrapper.spot import Spot
 
+###
+### Atomic Skills is a skill which requires its own policy and environment and does not depend on other atomic or composite skills
+###
+
+
+# Define Interface skill class
+class Skill:
```

Comment:
The point of having this class is to standardize our inputs/outputs of functions.
As a result, adding docstring once in the base class should be enough.

Reply:
If you find yourself adding docstrings per child class because the functions in the child classes are returning different things, then we should change the functions so that they return the same things (standardize).

Reply:
Removed all redundant docstrings from derived skill classes and kept them in base skill class.
All methods now have generic inputs/outputs.